mllib/src/main/scala/org/apache/spark/ml/param/shared/SharedParamsCodeGen.scala

```diff
@@ -44,6 +44,8 @@ private[shared] object SharedParamsCodeGen {
         " probabilities. Note: Not all models output well-calibrated probability estimates!" +
         " These probabilities should be treated as confidences, not precise probabilities",
         Some("\"probability\"")),
+      ParamDesc[String]("varianceCol", "Column name for the variance of prediction",
```

Comment:
Btw, I spoke with Xiangrui, and we decided it may be best to only turn one output column on by default.  If a user has a big Pipeline, they may run into a bunch of column name conflicts, forcing them to manually set every column.  So let's remove the default for this one and make it optional.